### PR TITLE
add course/teesheet id to booking result

### PR DESCRIPTION
**Pull request by michaeldfutch**
- Add course_id and teesheet_id fields to the json result
- Update test_booking to check for correct course_id

### CODE REVIEW

**Comment by michaeldfutch** [comment]
@doyouknolou I think you closed this pull request on accident. Re-opening it so that you can add a review, then I'll merge it.

File: test/test_sgg.py [hunk 1]
```diff
@@ -198,3 +197,11 @@ def test_secret_manager():
 def test_list_to_cloud_storage():
     response = cloud_utils.list_to_cloud_storage('sgg-test-bucket', JSON_LIST_EXAMPLE, 'test.json', timestamp=False)
     assert response == 'File test.json uploaded to sgg-test-bucket.'
+
+def test_get_customers():
+    username = 'mfutch78@gmail.com'
```
Comment (doyouknolou) [inline test/test_sgg.py:202]: i can continue to use your credentials if i run this on my own?

Comment (michaeldfutch) [reply test/test_sgg.py:202]: Yep, it works because it is using google secrets

**Review by doyouknolou — APPROVED** [review]
looks good to me!

**Pull request merged**